src/checkers/firefoxchecker.py

```diff
@@ -39,44 +39,50 @@
 
 
 class FirefoxChecker(Checker):
-    FIREFOX_RELEASE_INFO_URL = 'https://product-details.mozilla.org/1.0/firefox_versions.json'  # noqa: E501
-    FIREFOX_ARCHIVE_BASE_URL = 'https://archive.mozilla.org/pub/firefox/releases/{version}/'  # noqa: E501
-    FIREFOX_PLATFORM = 'linux-x86_64'
-    CHECKER_BROWSER_SOURCE_FILENAME = 'firefox.tar.bz2'
-    CHECKER_BROWSER_FILENAME_MATCH = r'firefox-.+\.tar\.bz2$'
-    CHECKER_BROWSER_DEFAULT_LANGUAGE = 'en-US'
-    CHECKER_TRANSLATIONS_FILENAME_MATCH = r'.+\.xpi$'
+    FIREFOX_RELEASE_INFO_URL = (
+        "https://product-details.mozilla.org/1.0/firefox_versions.json"  # noqa: E501
```

Comment:
```suggestion
        "https://product-details.mozilla.org/1.0/firefox_versions.json"
```

at which point it probably fits on one line again

Reply:
Will look into that tomorrow.

Reply:
It doesn't seem to affect the way Black formats it.